easybuild/easyconfigs/q/Q6/Q6-20180205-gompi-2019a.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'MakeCp'
+
+name = 'Q6'
+version = '20180205'
+local_commit = '5052ebe'
+
+homepage = 'https://github.com/qusers/Q6'
+description = """EVB, FEP and LIE simulator."""
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+github_account = 'qusers'
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['3743deac3c4f1251037b3128bdd0c850d23df2e219a7fddc5c801c82f07fefac']
+
+start_dir = 'src'
+
+buildopts = 'mpi COMP=gcc'
```

Comment:
did you consider also building the tools? it seems to work with `buildopts = 'tools COMP=gcc && make mpi COMP=gcc'`

Reply:
Done